drivers/watchdog/wdt_sam0.c

```diff
@@ -231,7 +234,11 @@ static int wdt_sam0_feed(struct device *dev, int channel_id)
 		return -EINVAL;
 	}
 
-	WDT_REGS->CLEAR.reg = WDT_CLEAR_CLEAR_KEY_Val;
+	if (!WDT_SYNCBUSY) {
```

Comment:
For consistency with the early-exit pattern above I'd rather see this as:
```
if (WDT_SYNCBUSY) {
    return -EAGAIN;
}

WDT_REGS->CLEAR.reg = WDT_CLEAR_CLEAR_KEY_Val;

return 0;
```

Reply:
+1

Reply:
+1

Reply:
done.